.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Why 16? It's neither the version we use now, nor the one we aim for

Reply:
okay down to 14